styles/main.css

```diff
@@ -0,0 +1,115 @@
+@import url("https://fonts.googleapis.com/css?family=Muli&display=swap");
+@import url("https://fonts.googleapis.com/css?family=Quicksand&display=swap");
+
+/* * {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+} */
+
+body {
+  width: 375px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 44px;
+  width: 375px;
+  left: 0;
+  top: 0;
+  border-radius: 0;
+  background-color: #191765;
+  padding: 0 24px;
+}
+
+main {
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+  padding: 80px 24px;
+  background-color: #191765;
+  background-image: url(../assests/eclipes.png);
+  background-position-x: -180px;
+  background-position-y: 40px;
+}
+
+.nav-menu {
+  cursor: pointer;
+}
+
+.logo a {
+  font-family: Roboto, sans-serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  color: #fff;
+  padding-left: 24px;
+  padding-right: 24px;
+  text-decoration: none;
+}
+
+.profile-container {
+  display: flex;
+  height: 450px;
+  width: 327px;
+  left: 24px;
+  top: 120px;
+  border-radius: 0;
+}
+
+.profile-container h1 {
+  font-family: Roboto, sans-serif;
+  font-size: 48px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 72px;
+  letter-spacing: 0.3700000047683716px;
+  text-align: center;
+  color: #7d73fc;
+}
+
+.profile-container h2 {
+  font-family: Roboto, sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 44px;
+  text-align: center;
+  color: #edecff;
+}
+
+p {
+  font-family: Roboto, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #c8c8d4;
+}
+
+button {
+  background-color: #191765;
+  border: #191765;
+}
+
+button:hover {
+  background-color: #aaa;
+}
+
+button:focus {
+  outline: none;
+  box-shadow: 0 0 0 3px lightskyblue;
+}
+
+button:active {
+  background-color: #333;
+  border-color: #333;
+  color: #eee;
+}
```

Comment:
- [ ] Good to see you've implemented the previous reviewer's required changes. However, the colours used are not the same as what is being specified on the Figma design. Kindly check and make the necessary changes.
![image](https://user-images.githubusercontent.com/16058470/121712056-2f92eb80-cad3-11eb-8419-856decc7f9c5.png)